tests/test_package_meta.py

```diff
@@ -13,3 +13,7 @@ def test_package_metadata_version():
         description_content_type="text/markdown",
     )
     assert pm3.metadata_version == "2.1"
+
+
+def test_hashable():
+    hash(PackageMetadata(name="test", version="1.0"))
```

Comment:
In the same spirit as the comment above
```suggestion
    assert hash(PackageMetadata(name="test", version="1.0")) == hash(PackageMetadata(name="test", version="1.0"))
    assert hash(PackageMetadata(name="test", version="1.0")) != hash(PackageMetadata(name="test", version="1.1"))
```